tests/test_utils/test_utils.py

```diff
@@ -0,0 +1,18 @@
+from openml.testing import TestBase
+import openml
+
+
+class OpenMLTaskTest(TestBase):
+    _multiprocess_can_split_ = True
+
+    def test_list_all(self):
+        list_datasets = openml.datasets.functions._list_datasets
+        datasets = openml.utils.list_all(list_datasets)
+
+        self.assertGreaterEqual(len(datasets), 100)
+        for did in datasets:
+            self._check_dataset(datasets[did])
+
+        # TODO implement these tests
```

Comment:
Sure you didn't forget these?

Reply:
you mean we should not verify the datasets?

Reply:
I mean there's still a TODO in the PR ;) 

(line 16)

Reply:
Ah you are right, I did not see what you meant because of the lines selected. The TODO seems already implemented.